tardis/model/base.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+from astropy import constants, units as u
+
+from tardis.model import density
+from tardis.util import quantity_linspace, InstanceDescriptorMixin
+
+
+class Radial1DModel(InstanceDescriptorMixin):
+    def __init__(self, velocity, homologous_density, abundance, t_inner,
+                 time_explosion, t_radiative=None, dilution_factor=None,
+                 v_boundary_inner=None, v_boundary_outer=None):
+        self.velocity = velocity
+        self.homologous_density = homologous_density
```

Comment:
Currently there is no way of accessing the non-adapted to `time_explosion` density array, i.e. `model.density` (which is now the same as `model.homologous_density`) will always return the scaled density. Do we need access to the starting density/density_0/however you call it?

Also, can we just name this density, instead of homologous_density?